index.js

```diff
@@ -1,59 +1,51 @@
 const express = require("express");
 const app = express();
-const port = 5000;
-
-const notes = [
-  {
-    id: 6,
-    title: "Josh",
-    isDone: true,
-  },
-  {
-    id: 2,
-    title: "Gabriel",
-    isDone: false,
-  },
-  {
-    id: 7,
-    title: "Srapion",
-    isDone: true,
-  },
-];
+const port = 9000;
+const db = require("./queries");
 
 app.use(express.json());
 
-app.get("/notes", (req, res) => {
-  res.send(notes);
+app.get("/", (request, response) => {
+  response.json({ info: "Node.js, Express, and Postgres API" });
 });
 
-app.get("/notes/:id", (req, res) => {
-  const finder = notes.find((note) => note.id == req.params.id);
-  if (finder) {
-    const temp = finder;
-    res.status(201);
-    res.send(finder);
-  }
-});
+app.get("/users", db.getUsers);
+app.post("/users", db.createUser);
+app.put("/users", db.updateUser);
+app.delete("/users", db.deleteUser);
```

Comment:
Since we're dealing with notes not users.
```suggestion
app.get("/notes", db.getNotes);
```